<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RCM Measurement GUI</string>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>1341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Status and Control</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PNA Connection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor Connection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pnaConnectionLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PNA Status</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="motorStatusLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor Status</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="systemStatusLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>System Status</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrateButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>105</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="measureDataButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>45</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editConfigButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>75</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reloadConfigButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>75</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopMeasurementButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>45</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Measurements</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor Position:</string>
     </property>
    </widget>
    <widget class="QLabel" name="motorPositionLabel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor Position</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="calibrationStatusLabel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration Status</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output File Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileNameLabel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>110</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QtCharts::QChartView" name="realChartView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QtCharts::QChartView" name="imagChartView">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>641</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logMessageWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>1341</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1362</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
